src/components/nav/Nav.js

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import logo from "../../assets/logo.png";
+import "./Nav.css";
+
+function Nav() {
+  const [profileData, setProfileData] = useState(null);
+
+  const fetchUserData = async () => {
+    try {
+      const response = await fetch(
+        "https://bootcamp-api.codeit.kr/api/sample/user"
+      );
+      if (response.ok) {
+        const userProfileData = await response.json();
+        setProfileData(userProfileData);
+      }
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    fetchUserData();
+  }, []);
+
+  return (
+    <nav>
+      <div class="gnb">
+        <a href="/">
+          <img src={logo} alt="홈으로이동" />
+        </a>
```

Comment:
r: 태은님의 리액트 앱은 SPA(싱글 페이지 애플리케이션)입니다. 하나의 html, js만 두고 실제 주소가 아닌 메모리 상에서 가상 주소를 변경하며 상태에 따라 동적으로 페이지 내의 컴포넌트를 렌더링하는 방식인데요, a 태그로 주소를 이동하게 되면 매번 브라우저가 페이지가 새로고침하면서 이전 상태를 잃게됩니다. react-router와 같은 라이브러리로 실제 주소가 아닌 가상 주소를 이동해주세요.

[[React] Link와 \<a\>태그의 href의 차이](https://gomgomkim.tistory.com/9)

```suggestion
        <Link to="/">
          <img src={logo} alt="홈으로이동" />
        </Link>
```